### DISCUSSION TITLE
Feature Request for MELCloud App on Homey - Enhanced Metrics for Mitsubishi Air-to-Water Heat Pump

### DISCUSSION THREAD

**Discussion by erwindek** [OlivierZal/com.melcloud#357] [Ideas]
Dear Oscar,

I hope this message finds you well. I am reaching out to express my appreciation for your work on the MELCloud app for the Homey platform, which I use to manage my Mitsubishi air-to-water heat pump. The app already provides a wealth of parameters that are incredibly useful for monitoring and managing the system. However, I would like to suggest a few enhancements that could further enrich the app's functionality and provide users with deeper insights into their heating systems.

Feature Requests:

1. Flow Metrics for the Central Heating (CH) Circuit and Domestic Hot Water (DHW) Circuit:
   - It would be highly beneficial to include metrics for the flow rate in both the central heating circuit and the domestic hot water circuit during heating. These metrics can help users better understand the efficiency and operation of their systems, particularly during the heating process.

2. Detection of Domestic Hot Water (DHW) Usage:
   - Another valuable addition would be a feature to detect when sanitary hot water is being tapped. This could involve monitoring changes in flow, temperature, or pressure that indicate DHW usage. Such a feature would allow users to track hot water consumption patterns and possibly optimize the system for energy savings.

These enhancements would not only improve user experience by providing more detailed system insights but also enable more precise control and optimization of energy usage.

I believe these features align with the app's goal of enhancing home energy management and would greatly appreciate your consideration of these suggestions. Thank you for your continuous efforts to improve the MELCloud app and for supporting Mitsubishi heat pump users.

Best regards,
Erwin

**Comment by OlivierZal** [comment]
Hi @erwindek,

Thank you very much for your feedback 😊

I can add new features to the MELCloud Homey app, provided that data can be retrieved from the MELCloud API.

To discuss further I invite to join the Homey Slack channel, will be easier: http://slack.athom.com

Kind regards,
Olivier
